components/constellation/constellation.rs

```diff
@@ -2063,7 +2063,11 @@ where
         };
         match request {
             FromScriptMsg::RequestAdapter(response_sender, options, ids) => match webgpu_chan {
-                None => warn!("Failed to send request adapter message, missing WebGPU channel"),
+                None => {
+                    if let Err(e) = response_sender.send(None) {
+                        return warn!("Failed to send request adapter message: {}", e);
```

Comment:
Does warn return `()`? If so can we keep return and warn in separate statements.

Reply:
Yes it returns `()`, I see that same approach is used in this file e.g:

https://github.com/servo/servo/blob/main/components/constellation/constellation.rs#L2027

Reply:
Then I guess it's alright.